index.md

```diff
@@ -1,11 +1,244 @@
+
+As a result, we have a critical need for a DP0.2-sized (minimum ~300 square degrees) data preview with real users working on LSSTCam data as an intermediate step before jumping to full DR1 size.
+
+**Recommendation:** A substantial DP2 release is needed to give us user behavior data on a genuine DR1 precursor while we still have the protection of the "preview" label so that we can explain to the community we are still learning. It should not be ruled out that an addition "preview" release will be required before we can stand behind something with the DR1 label.
+
+### Potential hybrid insufficiency
+
+Shortly after DP1 release, DP0.2 data was unavailable to data.lsst.cloud users (due to an S3DF compounded outage) for 9 days (DP1 catalogs were unavailable 16 hours longer than plannedq). In this light, the decision by Data Services to host DP1 images on Google Cloud may seem a fortuitous avoidance of a black swan event.
+
+What should not be lost is the reason for the decision to place DP1 next to the services in the first place.
+In the run-up to DP1, we found total end-to-end performance from the object store at S3DF to GGP was surprisingly poor, as well as highly variable.
+Additionally we had dropouts to communications to the Qserv REST API.
+Because of the DP1 schedule we did not have time to properly investigate these, and concerned that slow performance would seem unreasonable (and worrying) to users for such a small volume release, Data Services decided to host the DP1 images on Google Cloud.
+While this worked out reputationally, it means that the key operational model for a hybrid RSP (services on Google, data on SLAC) has not been proved by DP1, and remains a significant risk.
+
+Moreover, any trade-off is a two-edged sword; data at USDF means more network transfers, but putting more services at USDF so that can be next to the data (eg computational services) means fewer network transfers but more exposure to S3DF reliability.
+
+One advantage real-world DP1 usage gives us is a realistic user profile. We can use that to make decisions about whether to mitigate "pure hybrid" by having the most popular data products for the latest Data Release on GCP.
+
+**Recommendation:** No later than following DP0.2 release (but ideally earlier) prepare for a critical internal review of the pure hybrid plan.
+
+Data Services teams will start collecting data, conduct performance tests and calculate uptime statistics to support such a review.
+
+### User sign-ups
+
+The process for granting RSP accounts has the dubious merit of being simultaneously labor intensive, and insufficiently rigorous (and confusing to users but that is a more technical matter).
+
+Ironically, the requirement for manual intervention for user creation was an asset for DP1 - 2,000 extra users you weren't expecting can't suddenly show up because it takes several days to process account creation requests.
```

Comment:
```suggestion
Ironically, the requirement for manual intervention for user creation was an asset for DP1 - 2,000 extra users you weren't expecting can't suddenly show up on the services all at once, because it takes several days to process their account creation requests.
```